### PR TITLE
pull user authentication out of InternalServer class

### PR DESCRIPTION
**Pull request by brubbel**
User authentication code should not be interleaved with InternalServer.

Also: private_key from InternalServer -> Server.

### CODE REVIEW

File: opcua/server/internal_server.py [hunk 1]
```diff
@@ -339,6 +296,10 @@ def __init__(self, internal_server, aspace, submgr, name, user=User.Anonymous, e
         self.logger.info("Created internal session %s", self.name)
         self._lock = Lock()
 
+    @property
```
Comment (oroulet) [inline opcua/server/internal_server.py]: PEP8 says user_manager for class members

**Comment by oroulet** [comment]
changes look good

**Comment by brubbel** [comment]
done.

**Comment by oroulet** [comment]
when you are finished , remove the wip flag

**Comment by brubbel** [comment]
I think the page must be refreshed to see the updated title.

**Pull request merged**